src/__tests__/incidents/report/ReportIncident.test.tsx

```diff
@@ -48,229 +43,143 @@ describe('Report Incident', () => {
+      <ButtonBarProvider.ButtonBarProvider>
+        <Provider store={store}>
+          <Router history={history}>
+            <Route path="/incidents/new">
+              <titleUtil.TitleProvider>
+                <ReportIncident />
+              </titleUtil.TitleProvider>
+            </Route>
+          </Router>
+        </Provider>
+      </ButtonBarProvider.ButtonBarProvider>,
+    )
   }
+  test('type into department field', async () => {
+    setup([Permissions.ViewIncident, Permissions.ResolveIncident])
+    const depInput = await screen.findByPlaceholderText(/incidents\.reports\.department/i)
 
-  describe('title', () => {
-    it('should have called the useUpdateTitle hook', async () => {
-      await setup([Permissions.ReportIncident])
-      expect(titleUtil.useUpdateTitle).toHaveBeenCalledTimes(1)
-    })
-  })
-
-  describe('layout', () => {
-    it('should set the breadcrumbs properly', async () => {
-      await setup([Permissions.ReportIncident])
-
-      expect(breadcrumbUtil.default).toHaveBeenCalledWith([
-        { i18nKey: 'incidents.reports.new', location: '/incidents/new' },
-      ])
-    })
-
-    it('should have a date input', async () => {
-      const wrapper = await setup([Permissions.ReportIncident])
-
-      const dateInput = wrapper.findWhere((w) => w.prop('name') === 'dateOfIncident')
-
-      expect(dateInput).toHaveLength(1)
-      expect(dateInput.prop('label')).toEqual('incidents.reports.dateOfIncident')
-      expect(dateInput.prop('isEditable')).toBeTruthy()
-      expect(dateInput.prop('isRequired')).toBeTruthy()
-    })
-
-    it('should have a department input', async () => {
-      const wrapper = await setup([Permissions.ReportIncident])
-
-      const departmentInput = wrapper.findWhere((w) => w.prop('name') === 'department')
-
-      expect(departmentInput).toHaveLength(1)
-      expect(departmentInput.prop('label')).toEqual('incidents.reports.department')
-      expect(departmentInput.prop('isEditable')).toBeTruthy()
-      expect(departmentInput.prop('isRequired')).toBeTruthy()
-    })
-
-    it('should have a category input', async () => {
-      const wrapper = await setup([Permissions.ReportIncident])
-
-      const categoryInput = wrapper.findWhere((w) => w.prop('name') === 'category')
-
-      expect(categoryInput).toHaveLength(1)
-      expect(categoryInput.prop('label')).toEqual('incidents.reports.category')
-      expect(categoryInput.prop('isEditable')).toBeTruthy()
-      expect(categoryInput.prop('isRequired')).toBeTruthy()
-    })
+    expect(depInput).toBeEnabled()
+    expect(depInput).toBeInTheDocument()
 
-    it('should have a category item input', async () => {
-      const wrapper = await setup([Permissions.ReportIncident])
+    userEvent.type(depInput, 'Engineering Bay')
+    expect(depInput).toHaveDisplayValue('Engineering Bay')
+  })
 
-      const categoryInput = wrapper.findWhere((w) => w.prop('name') === 'categoryItem')
+  test('type into category field', async () => {
```

Comment:
Was the use of test vs it deliberate here? If so what is the thinking, just out of interest.

Reply:
No, that should be consistent.

Reply:
@nobrayner

Reply:
Fixed (to be merged)